mono/metadata/class-internals.h

```diff
@@ -941,6 +941,7 @@ typedef struct {
 	MonoClass *threadpool_wait_callback_class;
 	MonoMethod *threadpool_perform_wait_callback_method;
 	MonoClass *console_class;
+	MonoClass *bitconverter_class;
```

Comment:
We shouldn't add classes here unless they are accessed from multiple places during execution.